src/components/multi-action-button/multi-action-button.spec.tsx

```diff
@@ -170,6 +173,22 @@ describe("MultiActionButton", () => {
       });
     });
 
+    describe.each(["Enter", "Space"])(
+      "when the %s is pressed on a child button",
+      (key) => {
+        it("should call the onClick handler", () => {
+          mainButton.simulate("keydown", { key });
+
+          const button = wrapper
+            .find('[data-element="additional-buttons"]')
+            .find(Button);
+          button.at(0).simulate("click");
```

Comment:
Not sure if this is a typo- but why are we simulating `click`?. Isn't the whole point of this test to ensure that when Enter or Space is pressed, the onClick callback happens? By simulating click, we ensure that this test always passes, even when it shouldn't (I just tried removing the `if` statement you added in order to break the functionality - and sure enough, this test still passes.)

I understand that due to the rather limited way Enzyme's `.simulate` works, just replacing `"click"` with `"keydown"` here isn't going to trigger the click action - but triggering click manually afterwards to force it to pass isn't the right way of fixing this test. I expect we have to not use `.simulate` at all but use `document.dispatchEvent` or similar (as we do in some tests elsewhere).

Or maybe just refactor the tests to RTL :D